website/pages/docs/configuration/sentinel.mdx

```diff
@@ -25,12 +26,18 @@ A valid Vault Enterprise license is required for use of Sentinel policies.
 
 The sentinel stanza currently supports only one parameter, `additional_enabled_modules`.
 
-- `additional_enabled_modules` `(string array: [])`` - Specifies a list of
-  sentinel policy modules to enable above the default set.  For example,
-  adding "http" to this list indicates sentinel policies may import
-
-  ~> **Warning**: Care should be taken when enabling some modules which
-  could have performance and security implications in policies.  For
-  example, enabling "http" could cause your Vault servers to submit outbound
-  requests to arbitrary endpoints.  See [the Sentinel Documentation](https://docs.hashicorp.com/sentinel/imports/http/)
-  for more information.
+- `additional_enabled_modules` `(string array: [])`` - By default Vault supports
+  all of Sentinel's [standard imports](https://docs.hashicorp.com/sentinel/imports/)
+  except "http", which has performance and security implications.
+
+  This parameter specifies a list of modules to allow.  At this time "http"
+  is the only candidate, but in the future other Sentinel modules that may not
+  be included could be overridden here.
+  
+
+  ~> **Warning**: Care should be taken when enabling modules which
```

Comment:
For some reason when I tested this out the warning didn't format with the fancy box.

Reply:
Hmm.  How do we test this actually (ie render docs locally)

Reply:
See the `website/README.md`, but you can just `make` in that dir and then see it on `localhost:3000`.